### ISSUE TITLE
`ls` combined with passing username, password and registry fails

### ISSUE THREAD

**Issue by 030** [genuinetools/reg#38]
```
$ reg-linux-amd64 --insecure -u <user> -p <password> -r <registry> ls
No help topic for '<registry>'
ERROR: Job failed: exit code 1
```

while this works

```
reg-linux-amd64 -k ls
```

**Comment by jessfraz** [comment]
What is the real output of the fail

**Comment by 030** [comment]
```
FATA[0000] Get <url>/v2/: malformed auth challenge header: 'Basic realm="Registry Realm"' 
exit status 1
```

**Comment by jessfraz** [comment]
closing as this was fixed by #51
